.github/workflows/scripts/release-prepare-release.sh

```diff
@@ -41,7 +41,7 @@ if [ "${CANDIDATE_BETA}" != "" ]; then
 fi
 git push origin "${BRANCH}"
 
-gh pr create --title "[chore] Prepare release ${RELEASE_VERSION}" --body "
+gh pr create --title "[chore] Prepare release ${RELEASE_VERSION}" --label release:prepare --body "
```

Comment:
Can you also add a comment here stating why we add the label and referencing the workflow?